probe.go

```diff
@@ -801,6 +801,140 @@ func probeLinkMonitor(c FortiHTTP) ([]prometheus.Metric, bool) {
 	return m, true
 }
 
+func probeVirtualWanPerf(c FortiHTTP) ([]prometheus.Metric, bool) {
+	var (
+		mLink = prometheus.NewDesc(
+			"fortigate_virtualwan_healthcheck_status",
+			"Whether the link is up or not",
+			[]string{"vdom","sla", "interface", "state"}, nil,
```

Comment:
Is `sla` also called such somewhere in the Fortigate UI? We try to follow it in the labeling of the metrics.

Reply:
In the GUI it's called "Performance SLA" but in the api "Health check".
I thought "sla" is a nice compromise, because Performance SLA is a bit long.

Reply:
Perfect, I would have named it the same